doc/MANUAL.md

```diff
@@ -545,7 +545,7 @@ This option controls the frequency of updates for the CVE data from the National
 
 ### -n {json-nvd,json-mirror,api,api2}, --nvd {json-nvd,json-mirror,api,api2}
 
-This option selects how CVE data is downloaded from the National Vulnerability Database. The default `api` option uses the NVD CVE Retrieval API version 1.0. The `api2` option uses the later NVD CVE Retrieval API version 2.0. The results from this API are updated as quickly as the NVD website.
+This option selects how CVE data is downloaded from the National Vulnerability Database. The `api` option previously used the NVD CVE Retrieval API version 1.0, which is now deprecated. Using `--nvd api` will automatically failover to the `--nvd api2` option. The `api2` option uses the later NVD CVE Retrieval API version 2.0. The results from this API are updated as quickly as the NVD website.
```

Comment:
failover is the correct term but also kind of jargon-y (at least according to Microsoft: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/fail-over-fail-back-failover-failback ). You can add it to the dictionary in https://github.com/intel/cve-bin-tool/blob/main/.github/actions/spelling/allow.txt but it might be just as easy to change the text to say "switch"  as follows:

```suggestion
This option selects how CVE data is downloaded from the National Vulnerability Database. The `api` option previously used the NVD CVE Retrieval API version 1.0, which is now deprecated. Using `--nvd api` will automatically switch to the `--nvd api2` option. The `api2` option uses the later NVD CVE Retrieval API version 2.0. The results from this API are updated as quickly as the NVD database.
```

Reply:
Updated the documentation with suggested changes.